service_catalog/views/common.py

```diff
@@ -191,7 +195,37 @@ def dashboards(request):
     return render(request, 'service_catalog/common/dashboard.html', context=context)
 
 
-def request_comment(request, request_id, redirect_to_view, breadcrumbs):
+@permission_required_or_403('service_catalog.delete_request', (Request, 'id', 'request_id'))
```

Comment:
not sure if delete_request is the right permission to use here.
Could you take a look on how to create a new permission in Django?
If possible it would be interesting to have a "cancel_request" permission.

Reply:
Nevermind,
We'll refactor the perms when implementing correctly all RBAC.